styles/Index.module.css

```diff
@@ -0,0 +1,103 @@
+.heroBanner {
+    height:calc(40vh + 76px + 76px);
```

Comment:
I see 76px all over this file. What's the significance of that?

Reply:
76px is the height of the nav bar. I added the extra 76px in order to accommodate for smaller screens to display the banner correctly. Could be another way to make it more elegant.

Reply:
Ah I see, makes sense. Maybe just add a comment somewhere mentioning that.